python/src/core/Node.python.cpp

```diff
@@ -0,0 +1,157 @@
+// system includes
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+// local includes
+#include "GNDStk.hpp"
+
+// namespace aliases
+namespace python = pybind11;
+
+namespace core {
+
+  void wrapNode( python::module& module ) {
+
+    // type aliases
+    using Component = njoy::GNDStk::core::Node;
```

Comment:
It probably doesn't matter here, but Node is technically in GNDStk:: itself (for which core:: has a using).